Tests/Docker/Container/ContainerErrorHandlingTest.php

```diff
@@ -122,6 +122,7 @@ public function testOutOfMemory()
 
     public function testOutOfMemoryOverride()
     {
+        self::markTestSkipped('Unstable test');
```

Comment:
Vyhodil jsem to a zkusil jsem to pustit lokalne tak to bezelo vzdy. Na azure pipelines mi to spadlo .Tobe to tam padalo nekdy, nebo porad?

Nevim jestli by nebylo lepsi nechat ten test probehnout a handlovat az to kdyz to hodi tu jinou vyjimku s pameti a pak dat `markTestSkipped` nebo `marktTestIncomplete`

Reply:
pada to jenom na AZ pipeline když se to spustí poprvé a už me to mrdá to furt restartovat